graphql_compiler/tests/test_end_to_end.py

```diff
@@ -11,32 +12,41 @@
 EXAMPLE_GRAPHQL_QUERY = '''{
     Animal @filter(op_name: "name_or_alias", value: ["$wanted_name"]) {
         name @output(out_name: "name")
+        net_worth @filter(op_name: ">=", value: ["$min_worth"])
     }
 }'''
 
 
 class QueryFormattingTests(unittest.TestCase):
     def test_correct_arguments(self):
         wanted_name = 'Top Cat'
+        min_worth = Decimal('123456789.0123456')
```

Comment:
Would it be better to have the decimal argument test separate from the string argument test?

Reply:
Here we're not testing with a string argument at all -- we only test with a `Decimal` argument. For anything else, if Python knows how to turn it into a `Decimal` argument -- great, and if not we get an error.

Reply:
No, I meant the previous version of the test was only for `name_or_alias` string argument i.e. "Top Cat".
I was just wondering if it's better to test that separately from the decimal formatting.